src/app/reducers/user.reducer.ts

```diff
@@ -9,6 +10,8 @@ export interface State {
   user_profile: UserProfile;
   auth: any;
   selected_user_profile: UserProfile;
+  followers: Array<UserProfile>;
+  following: Array<UserProfile>;
```

Comment:
@ashish173 will change it in this story [link](https://www.pivotaltracker.com/n/projects/1927191/stories/137692253)